### PR TITLE
Portfolio: application deploy

### PR DESCRIPTION
**Pull request by mahdimahdawi**
- Modified README.md file
- Added project to github pages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ To get a local copy up and running follow these simple example steps.
 
 - GitHub: [@mahdimahdawi](https://github.com/mahdimahdawi)
 - LinkedIn: [Mahdi Mahdawi](https://linkedin.com/feed/)
-- Online link: https://mahdimahdawi.github.io
+- Github pages: [Mahdi-portfolio] https://mahdimahdawi.github.io
```
Comment (DammyShittu) [inline README.md]: - [ ] From the [readme template from Microverse](https://github.com/microverseinc/readme-template) we can see that the live link has its own section. Kindly move this link to its own section as well for better visibility and organization.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @mahdimahdawi,

✔️ Portfolio is properly deployed

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mahdimahdawi** [comment]
- Online link now available in Live section of README.md file.

**Comment by mahdimahdawi** [comment]
- Online link of portfolio project now available on live section of README.md file.

**Review by Reem-lab — APPROVED** [review]
# Hi @mahdimahdawi, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🔥🔥🔥💯💯


## Highlights
 - Good Commits messages ✅✅
 - Descriptive pull request and branch name ✅✅
 - No Linters error✅✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**